src/js/components/Form/Form.js

```diff
@@ -18,6 +18,22 @@ const defaultValidationResults = {
   infos: {},
 };
 
+const stringToArray = (string) => {
```

Comment:
This function is only specific to this string pattern - `arrayName[indexOfArray].ArrayObjectName`. Not sure, if in the future it will be used anywhere else.